TreeMaker/python/makeTreeFromMiniAOD_cff.py

```diff
@@ -3,23 +3,22 @@
 
 import FWCore.ParameterSet.Config as cms
 import sys,os
-
 def makeTreeFromMiniAOD(self,process):
 
     ## ----------------------------------------------------------------------------------------------
     ## ----------------------------------------------------------------------------------------------
     ## Preamble
     ## ----------------------------------------------------------------------------------------------
     ## ----------------------------------------------------------------------------------------------
-
+    
     # files to process
     import FWCore.PythonUtilities.LumiList as LumiList
-    from TreeMaker.TreeMaker.TMEras import TMeras
     process.maxEvents = cms.untracked.PSet(
         input = cms.untracked.int32(self.numevents)
     )
     process.source = cms.Source("PoolSource",
         fileNames = cms.untracked.vstring(self.readFiles),
+        secondaryFileNames=cms.untracked.vstring(self.readFiles_sidecar),
```

Comment:
please completely remove the most recent commit 0958a9c5406a5e17f90c8a3f93793af0aadc5f44 which seems to merge unrelated changes